gadget/install/install.go

```diff
@@ -166,11 +171,17 @@ func Run(model gadget.Model, gadgetRoot, kernelRoot, device string, options Opti
 		// matches what is on the disk, but sometimes there may not be a sector
 		// size specified in the gadget.yaml, but we will always have the sector
 		// size from the physical disk device
-		if err := makeFilesystem(&part, diskLayout.SectorSize); err != nil {
+		sp = perfTimings.StartSpan("make-filesystem", fmt.Sprintf("Create filesystem for %s", part.Node))
```

Comment:
we probably want a timing entry around newEncryptedDevice too, as this is where luksFormat happens, although I can't add a comment there as it seems to be outside of the context of the diff